flask_resources/loaders.py

```diff
@@ -40,11 +40,18 @@ def wrapper(self, *args, **kwargs):
         # Checking Content-Type is the responsibility of the loader
         loaders = self.resource.config.request_loaders
 
-        if request.content_type not in loaders:
+        request_content_type = request.content_type
+
+        if not request_content_type:
+            request_content_type = getattr(
+                self.resource.config, "default_content_type", None
+            )
+
+        if request_content_type not in loaders:
             raise InvalidContentType(allowed_mimetypes=loaders.keys())
 
         deserializer = select_deserializer(
-            self.resource_method, loaders[request.content_type]
+            self.resource_method, loaders[request_content_type]
```

Comment:
What happens if a content-type doesn't exists?

Reply:
It raises an `InvalidContentType` in this [line](https://github.com/inveniosoftware/flask-resources/pull/71/files#diff-de2cf242e5126989fce8235c111e296eR49)